src/components/collection/Collection.tsx

```diff
@@ -120,7 +127,12 @@ const Collection = ({ id }: Props) => {
           }}
           py={5}
         >
-          <Summary collection={collection} isLoading={isLoading} />
+          <Summary
+            collection={collection}
+            isLoading={isLoading}
+            // any types needs to be changed from sdk
```

Comment:
Okay, can you create the corresponding issue there And update the type accordingly ? We will need to release a version of sdk once done.